### PR TITLE
WIP: Set up CI using Travis

### PR DESCRIPTION
**Pull request by Dannnno**
- Add .travis.yml file
- Update README.md to show build status

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: objective-c  #Travis only recognizes Obj-C, Xcode will handle the fact that this is Swift
+
+# xcode and sdk versions
+osx_image: xcode7
+xcode_sdk: iphonesimulator9.0
+xcode_project: "APP/HMC Word Scroll/HMC Word Scroll.xcodeproj"
+xcode_scheme: "HMC Word Scroll"
+
+# Update any of our needed tools
+before_install:
```
Comment (hmc-cs-dobermiller) [inline .travis.yml]: This part is probably unnecessary, but is good practice

Comment (Dannnno) [inline .travis.yml:10]: Probably unnecessary, but good practice for how to update things using homebrew

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: objective-c  #Travis only recognizes Obj-C, Xcode will handle the fact that this is Swift
+
+# xcode and sdk versions
+osx_image: xcode7
+xcode_sdk: iphonesimulator9.0
```
Comment (Dannnno) [inline .travis.yml:5]: Eventually we can set up a matrix that will run against a lot of different sdks versions.

File: APP/HMC Word Scroll/HMC Word Scroll.xcodeproj/xcshareddata/xcschemes/HMC Word Scroll.xcscheme [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Dannnno) [inline APP/HMC Word Scroll/HMC Word Scroll.xcodeproj/xcshareddata/xcschemes/HMC Word Scroll.xcscheme:1]: Necessary for travis to know how to build

File: README.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/hmc-simplification/TextSimplification.svg)][https://travis-ci.org/hmc-simplification/TextSimplification]
```
Comment (Dannnno) [inline README.md:1]: This will put a pretty badge on the ReadMe.md

**Comment by Dannnno** [comment]
Superseded by #3

**Pull request closed without merging**